src/pow.cpp

```diff
@@ -24,20 +24,14 @@ unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHead
     if (pindexLast == NULL)
         return nProofOfWorkLimit;
 
-    const CBlockIndex* pindexBits = pindexLast;
     {
```

Comment:
It isn't necessary for this to be a block.

Reply:
Yes, also braces might help at line 25 and line 34-35 of this method.